tools/ncf.py

```diff
@@ -451,9 +465,16 @@ def generate_technique_content(technique_metadata):
 
   # Handle method calls
   for method_call in technique["method_calls"]:
+    method_name = method_call["method_name"]
     regex = re.compile(r'(?<!\\)"', flags=re.UNICODE )
     # Treat each argument of the method_call
     if 'args' in method_call:
+      for index, arg in enumerate(method_call['args']):
+        parameter = methods[method_name]["parameter"][index]
+        arg_constraint = parameter.get("constraints", {})
+        check = ncf_constraints.check_parameter(arg, arg_constraint)
```

Comment:
Wow, good catch. That's a very good point!

Should we implement the same behaviour as Rudder-web and replace "${.*}" and "$(.*)" with ".*" in the regex, do you think?